docs/articles/documentation/recipes/extract-reusable-test-code/create-helpers.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: docs
+title: Create Helpers
+permalink: /documentation/recipes/extract-reusable-test-code/create-helpers.html
+---
+# Create Helpers
+
+We recommend that you follow the [page model](use-page-model.md) pattern to extract reusable test code. This pattern allows you to abstract out both page structure and test logic.
+
+* [Use Page Model](use-page-model.md)
+
+However, if you need to extract only the helper functions, you can export them from a separate script file.
```

Comment:
If you need to extract only the helper functions, however, you can export them from a separate script file.